
int32 lesson_id
string lesson_name
bool changed


